src/components/Tooltip/index.test.js

```diff
@@ -16,13 +17,32 @@ const NoTooltipFixture = <Tooltip {...tooltipProps} open={false} />
 
 describe("Tooltip", () => {
   it("renders", () => {
-    mount(TooltipUpFixture)
-    mount(TooltipBottomFixture)
-    mount(TooltipLeftFixture)
-    mount(TooltipRightFixture)
+    const wrappers = [
+      mount(TooltipUpFixture),
+      mount(TooltipBottomFixture),
+      mount(TooltipLeftFixture),
+      mount(TooltipRightFixture),
+    ]
+
+    wrappers.forEach(wrapper => {
+      expect(wrapper).not.toHaveStyleRule("animation", /.*/)
```

Comment:
This test asserts I'm not adding the animation CSS property to components that haven't opted in. Makes sure this isn't a breaking change.